api/v1alpha1/clusterspiffeid_types.go

```diff
@@ -67,6 +67,12 @@ type ClusterSPIFFEIDSpec struct {
 	// administrative APIs. Extra care should be taken to only apply this
 	// SPIFFE ID to admin workloads.
 	Admin bool `json:"admin,omitempty"`
+
+	// Downstream indicates that the entry describes a downstream SPIRE server.
+	Downstream bool `json:"downstream,omitempty"`
+
+	// ParentID is the SPIFFE ID of the parent entry.
+	ParentID string `json:"parentId,omitempty"`
```

Comment:
Can we decouple the custom parent id from downstream and focus on just downstream with this pr? Parenting requires a little more thought.

Reply:
OK. Dropped that commit.